tests/integration/steps_democracy.go

```diff
@@ -203,7 +203,7 @@ func stepsDemocracy(consumerName string) []Step {
 					ValPowers: &map[validatorID]uint{
 						validatorID("alice"): 511,
 						// 1% of bob's stake should be slashed as set in config.go
```

Comment:
Remove this comment on tokens getting slashed

Reply:
See https://github.com/cosmos/interchain-security/pull/692/commits/238a69f4c62a62246591ab45234d5891c1a56a3b